src/views/certifications/views/landing/components/signup-form-area/components/newsletter-signup-form/marketo-form-data.module.css

```diff
@@ -0,0 +1,108 @@
+.formWrapper {
+	width: 100%;
+}
+
+.form {
```

Comment:
This file is pretty messy, but im not sure another way to style out this marketo form component without these hacks, feels _okay_ as a first spike 🤷🏻